src/styles/blocks/star.css

```diff
@@ -0,0 +1,22 @@
+
+.stars {
+  display: flex;
+  width: 100px;
+}
+
+.stars__star {
+  background-image: url(../../images/star-active.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-of-type {
+  margin-right: 1px;
+}
+
+.stars--4 .stars__star:nth-of-type(n+5) {
```

Comment:
keep it operational - add modifiers for `.stars--1`, `.stars--2` etc., so if you change `stars--4` modifier to `stars--2` or other - number of selected stars should change correctly